<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QmitkUSCombinedModalityEditWidget</class>
 <widget class="QWidget" name="QmitkUSCombinedModalityEditWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>404</width>
    <height>418</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>350</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Identifier:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="nameLineEditLabel">
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="nameLineEdit">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="vendorLineEditLabel">
       <property name="text">
        <string>Vendor</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="vendorLineEdit">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="usDevicesServiceListLabel">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Ultrasound Device:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="ultrasoundDeviceLabel">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="trackingDevicesServiceListLabel">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Tracking Device:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="trackingDeviceLabel">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_4">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Calibrations:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QTableView" name="calibrationsTable">
        <property name="selectionMode">
         <enum>QAbstractItemView::NoSelection</enum>
        </property>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>80</number>
        </attribute>
        <attribute name="horizontalHeaderMinimumSectionSize">
         <number>15</number>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="calibrationsLoadButton">
          <property name="toolTip">
           <string>Load Calibrations from File System</string>
          </property>
          <property name="text">
           <string>Load</string>
          </property>
          <property name="icon">
           <iconset resource="../../../resources/USNavigation.qrc">
            <normaloff>:/USNavigation/document-open.png</normaloff>:/USNavigation/document-open.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="calibrationsSaveButton">
          <property name="toolTip">
           <string>Write Calibrations to File System</string>
          </property>
          <property name="text">
           <string>Write</string>
          </property>
          <property name="icon">
           <iconset resource="../../../resources/USNavigation.qrc">
            <normaloff>:/USNavigation/document-save.png</normaloff>:/USNavigation/document-save.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="saveButton">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
       <property name="icon">
        <iconset resource="../../../resources/USNavigation.qrc">
         <normaloff>:/USNavigation/accept.png</normaloff>:/USNavigation/accept.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="icon">
        <iconset resource="../../../resources/USNavigation.qrc">
         <normaloff>:/USNavigation/restart.png</normaloff>:/USNavigation/restart.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../resources/USNavigation.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>saveButton</sender>
   <signal>clicked()</signal>
   <receiver>QmitkUSCombinedModalityEditWidget</receiver>
   <slot>OnSaveButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>104</x>
     <y>347</y>
    </hint>
    <hint type="destinationlabel">
     <x>201</x>
     <y>184</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>QmitkUSCombinedModalityEditWidget</receiver>
   <slot>OnCancelButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>299</x>
     <y>347</y>
    </hint>
    <hint type="destinationlabel">
     <x>201</x>
     <y>184</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>calibrationsSaveButton</sender>
   <signal>clicked()</signal>
   <receiver>QmitkUSCombinedModalityEditWidget</receiver>
   <slot>OnCalibrationsSaveButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>201</x>
     <y>348</y>
    </hint>
    <hint type="destinationlabel">
     <x>201</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>calibrationsLoadButton</sender>
   <signal>clicked()</signal>
   <receiver>QmitkUSCombinedModalityEditWidget</receiver>
   <slot>OnCalibrationsLoadButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>109</x>
     <y>337</y>
    </hint>
    <hint type="destinationlabel">
     <x>201</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>SignalAborted()</slot>
  <slot>OnSaveButtonClicked()</slot>
  <slot>OnCancelButtonClicked()</slot>
  <slot>OnCalibrationsSaveButtonClicked()</slot>
  <slot>OnCalibrationsLoadButtonClicked()</slot>
 </slots>
</ui>
